src/components/ListItem/ListItem.tsx

```diff
@@ -120,6 +117,8 @@ const useStyles = (isRecent: boolean) =>
       overflow: 'hidden',
       whiteSpace: 'nowrap',
       maxWidth: '100%',
+      marginLeft: 6,
+      marginTop: 1,
```

Comment:
Why all these margins? `appName` classname is also used for styling url part of `ListItem` so it looks strange now
<img width="630" alt="Screen Shot 2020-09-11 at 15 21 39" src="https://user-images.githubusercontent.com/16701409/92930556-83887b00-f442-11ea-9408-0e4d464a0128.png">